newbank/server/Customer.java

```diff
@@ -6,12 +6,16 @@
 public class Customer {
 
 	private ArrayList<Account> accounts;
+	private ArrayList<Loan> loans;
 	private CustomerID customerID;
 	private String Name;
 
 	public Customer(String customerName, String userName, String password) {
 		accounts = new ArrayList<>();
-		customerID = new CustomerID(customerName, userName, password);
+
+		loans = new ArrayList<>();
+		customerID = new CustomerID(customerName, userName, password, iban);
```

Comment:
When building the project, I got an error due to iban. As I understand we don't have to add iban to create a new CustomerID.